yarn.lock

```diff
@@ -2060,6 +2060,11 @@ dot-prop@^5.2.0:
   dependencies:
     is-obj "^2.0.0"
 
+dotenv@^10.0.0:
```

Comment:
See other remark. I really like to keep this lib as clean as possible in terms of deps

Reply:
Removed!